### ISSUE TITLE
npm install gives error

### ISSUE THREAD

**Issue by caulagi** [luisrudge/generator-simple-react-browserify#1]
I got the following error when trying to install using npm -

```
eact » sudo npm install -g generator-simple-react-browserify                                                                  /tmp/react 1 ↵ 
npm ERR! 404 404 Not Found: generator-simple-react-browserify
npm ERR! 404 
npm ERR! 404 'generator-simple-react-browserify' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

npm ERR! System Linux 3.13.0-45-generic
npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "generator-simple-react-browserify"
npm ERR! cwd /tmp/react
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR! code E404
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /tmp/react/npm-debug.log
npm ERR! not ok code 0
react »
```

**Comment by luisrudge** [comment]
Damnnnnnnnnnnn! I forgot to publish to npm. I'm so embarrassed. This is published now! https://www.npmjs.com/package/generator-simple-react-browserify

**Comment by luisrudge** [comment]
@caulagi thanks for the heads up

**Comment by luisrudge** [comment]
ops. it should have closed #2
